<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pump_title_info</class>
 <widget class="QWidget" name="Pump_title_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1" colspan="2">
      <widget class="QGroupBox" name="ElectricControl">
       <property name="title">
        <string>Электронное управление</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="l_ac_dc">
          <property name="text">
           <string>Тип напряжения питания</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="l_voltage">
          <property name="text">
           <string>Напряжение питания, В</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="AcDc">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Постоянное</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Переменное</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="Voltage">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>110</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>220</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QGroupBox" name="Section1">
       <property name="title">
        <string>Секция №1</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="l_work_volume">
          <property name="text">
           <string>Рабочий объем, см³/об</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="Pressure">
          <property name="title">
           <string>Давление в напорной полости, бар:</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="l_min_pressure">
             <property name="text">
              <string>Минимальное</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="l_nom_pressure">
             <property name="text">
              <string>Номинальное</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="l_max_pressure">
             <property name="text">
              <string>Максимальное</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="MinPressure">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="NomPressure">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="MaxPressure">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="WorkVolume">
          <property name="toolTip">
           <string>0...355 см³/об</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="SerNo"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="l_model">
       <property name="text">
        <string>Модель гидронасоса</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="Spin">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Левое</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Правое</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="l_volume_kpd">
       <property name="text">
        <string>Объемный КПД насоса</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="l_ser_no">
       <property name="text">
        <string>Серийный номер</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QGroupBox" name="HydroControl">
       <property name="title">
        <string>Гидравлическое управление</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="l_max_control_pressure">
          <property name="text">
           <string>Максимальное давление
в канале управления, бар</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="MaxControlPressure">
          <property name="toolTip">
           <string>0...100 бар</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1" rowspan="4" colspan="2">
      <widget class="QGroupBox" name="Frequency">
       <property name="title">
        <string>Частота вращения насоса(ов), об/мин:</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="l_min_freq">
          <property name="text">
           <string>Минимальная</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="l_nom_freq">
          <property name="text">
           <string>Номинальная</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="l_max_freq">
          <property name="text">
           <string>Максимальная</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="MinFrequency">
          <property name="toolTip">
           <string>200...4000 об/мин</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="NomFrequency">
          <property name="toolTip">
           <string>200...4000 об/мин</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="MaxFrequency">
          <property name="toolTip">
           <string>200...4000 об/мин</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="l_user">
       <property name="text">
        <string>Ответственный</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="User"/>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="l_drainage">
       <property name="text">
        <string>Дренаж насоса при
номинальном давлении и
номинальном расходе, л/мин</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <widget class="QGroupBox" name="Section2">
       <property name="title">
        <string>Секция №2</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="l_work_volume_2">
          <property name="text">
           <string>Рабочий объем, см³/об</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="Pressure_2">
          <property name="title">
           <string>Давление в напорной полости, бар:</string>
          </property>
          <layout class="QFormLayout" name="formLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="l_min_pressure_2">
             <property name="text">
              <string>Минимальное</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="l_nom_pressure_2">
             <property name="text">
              <string>Номинальное</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="l_max_pressure_2">
             <property name="text">
              <string>Максимальное</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="MinPressure_2">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="NomPressure_2">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="MaxPressure_2">
             <property name="toolTip">
              <string>0...450 бар</string>
             </property>
             <property name="toolTipDuration">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="WorkVolume_2">
          <property name="toolTip">
           <string>0...355 см³/об</string>
          </property>
          <property name="toolTipDuration">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="l_spin">
       <property name="text">
        <string>Направление вращения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="Model">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QDoubleSpinBox" name="VolumeKPD">
       <property name="toolTip">
        <string>0...1</string>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.950000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLineEdit" name="Drainage">
       <property name="toolTip">
        <string>0.7...90 л/мин</string>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="AnsverBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
